_dprhtml/js/keyboard_shortcuts.test.js

```diff
@@ -0,0 +1,29 @@
+import userEvent from "@testing-library/user-event";
+
+import { DprViewModel } from "./dprviewmodel";
+
+import "./keyboard_shortcuts";
+
+const resetSettings = jest.fn();
+
+const MockOtherDialogsViewModel = jest.fn().mockReturnValue({
+  resetSettings,
+});
+
+beforeEach(() => {
+  window.navigationFeatureName = "";
+  window.__dprViewModel = new DprViewModel();
+  window.__otherDialogsViewModel = new MockOtherDialogsViewModel();
```

Comment:
The closer we move to using modern JS modules, the less global setup we will have to do in our tests.